upup/models/cloudup/resources/addons/dns-controller.addons.k8s.io/k8s-1.6.yaml.template

```diff
@@ -44,6 +44,14 @@ spec:
         - name: {{ $name }}
           value: {{ $value }}
 {{ end }}
+{{- end }}
+{{- if eq .CloudProvider "digitalocean" }}
+        env:
+        - name: DIGITALOCEAN_ACCESS_TOKEN
+          valueFrom:
+            secretKeyRef:
+              name: digitalocean
+              key: access-token
```

Comment:
We may want to create a separate secret for DNS than for cloud-controller-manager, even if they have the same values, to pave the way for a future where they can be different with different permissions.  But for now YAGNI!